modules/install-osp-deploy-dualstack.adoc

```diff
@@ -164,4 +164,10 @@ ipv6.addr-gen-mode=0
 ----
 
 After you create and edit the file, reboot the installation host.
+====
+
+[NOTE]
+====
+The `ip=dhcp,dhcp6` kernel argument, which is set on all of the nodes, results in a single Network Manager connection profile that is activated on multiple interfaces simultaneously.
+Because of this behavior, any additional network will have the same connection enforced with an identical UUID. If you need an interface-specific configuration, create a new connection profile for that interface so that the default connection is no longer enforced on it.
```

Comment:
For some reason, we don't like the word _will_.

```suggestion
Because of this behavior, any additional network has the same connection enforced with an identical UUID. If you need an interface-specific configuration, create a new connection profile for that interface so that the default connection is no longer enforced on it.
```